push_docker.sh

```diff
@@ -1,42 +1,42 @@
+#! /bin/bash 
+
 DOCKER_IMAGE_FILE=$1
 REGISTRY_SERVER=$2
-REGISTRY_PORT=$3
```

Comment:
If you remove this parameter, you will break backward compatibility.

Reply:
Mostly PORT is not required. Where required, it could be given as part of REGISTRY_SERVER (acs.cc.com:5000)

Can we use this opportunity to cleanup ?

Reply:
Since it is already used, it will break.

---
In reply to: [402509324](https://github.com/Azure/sonic-buildimage/pull/4337#discussion_r402509324) [](ancestors = 402509324)

Reply:
We can fix it in the job.

Reply:
There is nothing wrong here, and you should keep it.

---
In reply to: [402698136](https://github.com/Azure/sonic-buildimage/pull/4337#discussion_r402698136) [](ancestors = 402698136,402509324)

Reply:
We are keeping it